docs/user/openstack/README.md

```diff
@@ -646,6 +646,48 @@ If you ran the installer with a [custom CA certificate](#self-signed-openstack-c
 oc edit configmap -n openshift-config cloud-provider-config
 ```
 
+### Enabling Octavia for Load Balancer Services
+
+OpenShift supports creating load balancer service types, and ingress controllers with load balancers as a backend using octavia as a day 2 operation. However, there are a few known issues to be aware of:
+* only TCP traffic is supported
+* the floating IPs attached to your Octavia load balancers will not be deleted if you delete your cluster
+* `manage-security-groups` will not work for non-admin tenants
+* `loadBalancerSourceRanges` is currently [not supported](https://bugzilla.redhat.com/show_bug.cgi?id=1980748)
+
+Enabling this service should only be done with non-kuryr deployments as a day 2 operation. To do this, edit the cloud provider config:
+
+```sh
+oc edit configmap -n openshift-config cloud-provider-config
+```
+
+If you are using the `amphora` driver, then you should add this to your cloud provider config:
+
+```
+[LoadBalancer]
+use-octavia = true
+lb-provider = amphora
+```
+
+If you are using the `ovn` driver, then you should add this to your cloud provider config:
+
+```
+[LoadBalancer]
+use-octavia = true
+lb-provider = ovn
+lb-method = SOURCE_IP_PORT
+```
+
+If you have multiple external networks, you need to set `floating-network-id` in your cloud provider config to the one you want FIPs to be created in:
+
+```
+[LoadBalancer]
+use-octavia = true
+lb-provider = amphora
+floating-network-id = < network uuid >
+```
+
+If you are using the OVN driver for Octavia, you will also need to modify the `_ingress_services_tcp` security group rules in the master and worker security groups to have a `remote ip prefix = 0.0.0.0/0`.
```

Comment:
The leading underscore feels a bit strange, I suggest removing it:
```suggestion
If you are using the OVN driver for Octavia, you will also need to modify the `ingress_services_tcp` security group rules in the master and worker security groups to have a `remote ip prefix = 0.0.0.0/0`.
```

Reply:
Its because they are named master_ingresss_services_tcp and worker_ingress_services_tcp. I'll just enumerate them like that instead

Reply:
The security group rules don't have names in openstack though, that's only a helper for us in terraform:
```
$ openstack security group rule list --long
+--------------------------------------+-------------+-----------+---------------+-------------+-----------+--------------------------------------+--------------------------------------+
| ID                                   | IP Protocol | Ethertype | IP Range      | Port Range  | Direction | Remote Security Group                | Security Group                       |
+--------------------------------------+-------------+-----------+---------------+-------------+-----------+--------------------------------------+--------------------------------------+
| 004530fd-6d62-4e81-8254-f6d1eb6f5b7b | None        | IPv6      | ::/0          |             | egress    | None                                 | f2db412f-3e6e-4864-8554-852791d90592 |
| 0151c4e1-b5db-4e80-988a-fbded1d947d7 | tcp         | IPv4      | 172.16.0.0/24 | 10259:10259 | ingress   | None                                 | 1776e4d5-6f51-44a2-b09e-91b875b80aff |
| 01ae1631-92b4-4e8b-9041-00e2971feadd | None        | IPv4      | 0.0.0.0/0     |             | egress    | None                                 | 98b7fa25-a88e-4f85-8c22-e696d8bac093 |
| 03a4b18e-8203-4b97-952c-9abb8e992a13 | 112         | IPv4      | 10.0.0.0/16   |             | ingress   | None                                 | f2db412f-3e6e-4864-8554-852791d90592 |
...
```